app/res/layout/draw_layout.xml

```diff
@@ -20,6 +20,7 @@
         <Button
             android:id="@+id/btnFinishDraw"
             android:layout_width="wrap_content"
+            android:enabled="false"
             android:layout_height="wrap_content"/>
 
         <LinearLayout
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/fd97cc14c2cf8a94c905831da7cd406d17d4705b/app/res/layout/draw_layout.xml#L26"></span><code>LinearLayout-3</code> can overlap <code>@id/btnFinishDraw</code> if @id/btnFinishDraw, LinearLayout-3 grow due to localized text expansion</td>
    </tr>
  </tbody>
</table>